test/components/CalendarDay_spec.jsx

```diff
@@ -84,6 +92,10 @@ describe('CalendarDay', () => {
       handleDayMouseUpSpy = sinon.spy(CalendarDay.prototype, 'handleDayMouseUp');
     });
 
+    afterEach(() => {
+      sinon.restore();
```

Comment:
@spikebrehm There's one here: https://github.com/airbnb/react-dates/blob/master/test/_helpers/restoreSinonStubs.js, but it didn't seem to do the trick.